api/views.py

```diff
@@ -124,11 +127,13 @@ def create_redflag():
                        status, createdOn, images, videos
                        )
     error = Validators.validate_inputs(redflag)        
-    exists = check_incident_exist(title)
+    exists = [redflag for redflag in incidents]
     if error != None:
         return jsonify({'Error': error, 'status': 400}), 400
     if exists:
```

Comment:
I do not see how this eliminates adding duplicate records. This line of code is true if the list of incidents is empty and false if it is not. [Please kindly refer to the issue I opened on your repo for more insight](https://github.com/bekeplar/iReporter/issues/22) on how to deal with duplicate records.